setup.py

```diff
@@ -61,6 +61,13 @@
         'pyld>=0.7.1',
     ],
     'tests': tests_require,
+    # Marshmallow v3.x drops support for Python 2
+    ':python_version<"3.0.0"': [
+        'marshmallow>=2.15.2,<3.0.0',
```

Comment:
@kprzerwa I noticed that in `invenio-rest` you've specified [`marshmallow v2.15.1` as lowest](https://github.com/inveniosoftware/invenio-rest/commit/e4acef161f0f4b7291d88d262924089b67d30680#diff-2eeaed663bd0d25b7e608891384b7298R40). Since `webargs 5.x` requires [`marshmallow v2.15.2` as lowest](https://github.com/marshmallow-code/webargs/blob/2cf69ec85f80f97eb61848fc1162596e110096b5/setup.py#L6), I'm curious if you had a specific reason in mind?

Reply:
@slint will you fix it in invenio-rest?

Reply:
:+1: I checked with @kprzerwa and she said it was like that there. I'm making a PR

Reply:
Actually, `webargs` is not used in `invenio-rest`, not sure if it should be a dependency...